src/lib.ts

```diff
@@ -212,3 +272,30 @@ export function digitizedMonstersRemaining(): number {
     SourceTerminal.getDigitizeMonsterCount()
   );
 }
+
+export function toasterGazeFor(location: Location): () => void {
```

Comment:
My understanding of the orb is that toaster gaze is only *half* of the reset process.  frazazel explains this prety clearly [in discord](https://discord.com/channels/466605739838930955/474736566246572042/1058077958394368093).

```

Reply:
Update on this -- per our convo I totally missed that the second half is already baked into this implementation!  (e.g. right now this script gazes EVERY turn, and if a prediction is bad it simply unequips orb which will functionally increment the age of the bad prediction and then the next gaze will reset).  🎉

Reply:
Just implemented ponder-caching, which I think should help. Gazecaching will be hard, though.